src/components/views/messages/TextualBody.js

```diff
@@ -128,6 +132,13 @@ export default class TextualBody extends React.Component {
         }
     }
 
+    _addCodeElement(pre) {
+        const code = document.createElement("code");
+        code.innerHTML = pre.innerHTML;
```

Comment:
Let's avoid using `innerHTML` if we can, see https://stackoverflow.com/a/6938316/486978 on how you can wrap an element.